public/app/features/dashboard/utils/getPanelMenu.ts

```diff
@@ -278,5 +286,51 @@ export function getPanelMenu(
     });
   }
 
+  const { extensions } = getPluginExtensions({
+    placement: GrafanaExtensions.DashboardPanelMenu,
+    context: createExtensionContext(panel, dashboard),
+  });
+
+  for (const extension of extensions) {
+    if (isPluginExtensionLink(extension)) {
+      subMenu.push({
+        text: truncateTitle(extension.title, 25),
+        href: extension.path,
+      });
+      continue;
```

Comment:
**Nit:** do we need the `continue;` statement here?

Reply:
I've removed it as I don't think it's needed either and local checks and tests pass. Maybe @mckn knows something we don't?

Reply:
Nah, I think this should be fine (I guess previously there was some code below it which served as an "else" block, and the `continue;` was used to "bail out early". Will resolve this - @mckn please reopen if you disagree.